tox.ini

```diff
@@ -0,0 +1,5 @@
+[tox]
+envlist = py27, py34
```

Comment:
Perhaps, it would be a good idea to add py34 to envlist in some later pull request, since the package does _not_ work in python3 yet? If I merge this right now, we will end up with master, that fails `make test`